### ISSUE TITLE
kendo-angular-layout@1.0.8 package seems corrupted

### ISSUE THREAD

**Issue by jackjoy** [telerik/kendo-angular#1062]
```
@progress/kendo-angular-layout@1.0.8 seems corrupted,it does not have right files.

```

**Comment by tsvetomir** [comment]
Thanks for the heads-up, will investigate.

**Comment by tsvetomir** [comment]
Fixed in v1.0.9, apologies for the caused inconvenience.

We managed to identify the cause (see telerik/kendo-angular#1063). No other packages are affected.
